Tests/PubNubTests/Networking/Routers/SubscribeRouterTests.swift

```diff
@@ -774,6 +774,57 @@ extension SubscribeRouterTests {
   }
 }
 
+// MARK: - Error Handling
+extension SubscribeRouterTests {
+  func testInvalidJSONResponse() {
+    let corruptBase64Response = "eyJ0Ijp7InQiOiIxNTkxMjE4MzQ0MTUyNjM1MCIsInIiOjF9LCJtIjpbeyJhIjoiMyIsImYiOjUxMiwicCI6eyJ0IjoiMTU5MTIxODM0NDE1NTQyMDAiLCJyIjoxfSwiayI6ImRlbW8tMzYiLCJjIjoic3dpZnRJbnZhbGlkSlNPTi7/IiwiZCI6ImhlbGxvIiwiYiI6InN3aWZ0SW52YWxpZEpTT04uKiJ9XX0="
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line should be 120 characters or less: currently 266 characters](https://app.codacy.com/manual/PubNub/swift/pullRequest?prid=5635978)